docs/products/opensearch/howto/migrate-snapshot-data-opensearch.md

```diff
@@ -0,0 +1,339 @@
+---
+title: Migrate data to Aiven for OpenSearch® using snapshots
+sidebar_label: Migrate snapshot data
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Aiven for OpenSearch enables you to restore data from external OpenSearch or Elasticsearch snapshots, enabling seamless migration from third-party repositories.
+
+Aiven supports snapshot restoration from Google Cloud Storage (GCS), Amazon S3, and
+Microsoft Azure. Additionally, you can restore data from Oracle Cloud Infrastructure
+(OCI) or other S3-compatible repositories using the S3 repository type.
+
+## Supported cloud providers
+
+Aiven for OpenSearch supports restoring snapshots from the following cloud providers:
+
+- Google Cloud Storage (GCS)
+- Amazon S3
+- Microsoft Azure
+- S3-compatible services. For example, Oracle Cloud Infrastructure
+
+## Prerequisites
+
+Before you begin, ensure that:
+
+- You have an active Aiven account with a configured project.
+- A target Aiven for OpenSearch service within your project.
+- Your OpenSearch or Elasticsearch snapshot is from version 7.10.2 or earlier and is
+  compatible with your target service.
+- Enable the [feature preview](/docs/platform/howto/feature-preview) to try
+  migrating external snapshots.
```

Comment:
This bullet doesn't work with with your introduction sentence.

Reply:
Thanks! This feature is behind ACL. Update the LA tag.